core/src/components/range/range.tsx

```diff
@@ -353,27 +361,52 @@ export class Range implements ComponentInterface {
   render() {
     const { min, max, step, ratioLower, ratioUpper } = this;
 
-    const barL = `${ratioLower * 100}%`;
-    const barR = `${100 - ratioUpper * 100}%`;
+    const barStart = `${ratioLower * 100}%`;
+    const barEnd = `${100 - ratioUpper * 100}%`;
+
+    const isRTL = document.dir === 'rtl';
+    const start = isRTL ? 'right' : 'left';
+    const end = isRTL ? 'left' : 'right';
 
     const ticks = [];
     if (this.snaps) {
       for (let value = min; value <= max; value += step) {
         const ratio = valueToRatio(value, min, max);
-        ticks.push({
+
+        const t: any = {
           ratio,
           active: ratio >= ratioLower && ratio <= ratioUpper,
-          left: `${ratio * 100}%`
-        });
+        };
+
+        t[start] = `${ratio * 100}%`;
+
+        ticks.push(t);
       }
     }
 
+    const tickStyle = (t: any) => {
```

Comment:
Can you change `t` to something more descriptive? Also is there a type available for this?

(can you do this to the `t` here and the `t` on line 376?)

Reply:
Ok, but there's no type here. Should I comment the code?

Reply:
if there's no type don't worry about it.